### ISSUE TITLE
USB Cable Tidy Door Hanger

### ISSUE THREAD

**Issue by nelsonic** [dwyl/learn-cad-cam-cnc#87]
Tired of having USB Cables disorganised in a Kallax box. 
Today's mini-project is a storage system for USB cables.

# Todo

+ [x] Measure cellar door width [internal]
+ [x] Measure USB Cable head & tail
+ [x] Research Average Cable Length (e.g. 3 Foot? 1M? 2M?)
+ [x] Decide how to store longer cables. e.g: 3m (we have a few of these)
+ [x] Create basic design we can cut from scrap materials (10mm birch plywood)
+ [x] Cut prototype on Laser
+ [ ] Install on the cellar door [inside]

I _estimate_ that we can get this done in **`T25m`** ⏳

**Comment by nelsonic** [comment]
Made a quick design:

![image](https://user-images.githubusercontent.com/194400/213587854-e2793529-c200-4723-88b0-f4055db6ed87.png)

Laser-cutting it _now_! ⚡ 
_Really_ want to tidy all the cables ASAP so that they aren't a mess in a box/bag. 🙏 
It's quite late so I don't think I will _mount_ it to the cellar door _today_ ... 💭 
But should be quick to mount tomorrow and sort/store the cables. ⏳

**Comment by nelsonic** [comment]
Laser cutting complete:

![B2B5CEBF-7C47-443F-B73B-98209E1BCDE4](https://user-images.githubusercontent.com/194400/213592085-03d7c47d-acc7-44c1-a772-6c7a5398ab3c.jpeg)

QR code works:

![447E1B0A-705E-4DFE-BB8A-A3CD6EAC5779](https://user-images.githubusercontent.com/194400/213592112-2eb42b36-cd0b-49f1-8504-0085d1322d88.jpeg)

Going to glue it now so it’s ready to mount tomorrow during the day when I take a 5min break from work. 👌

**Comment by nelsonic** [comment]
Glue-up/assembly complete:

![6BB8D9FC-D8F0-4DA6-B86E-63B09E34A7D4](https://user-images.githubusercontent.com/194400/213593768-6a32d146-e761-45c9-9716-b7c2bac75c62.jpeg)

**Comment by nelsonic** [comment]
Glue was already dry enough to mount it so I did:

![06A06A41-1DEE-4014-A9E9-0D52A2F27AE8](https://user-images.githubusercontent.com/194400/213594819-d588bdbc-2c01-48c9-9b47-08663ec67093.jpeg)

2 screws into a wooden door.
Virtually silent with a hand screwdriver.
Added a bunch of cables. 
Prototype works exactly as I planned.

can easily group like cables in a single slot and place infrequently used cables further. 
29 slots x 30mm is enough for over 100 cables. Never searching for another cable again!! 👌

And looking at the door from the outside of the cellar you wouldn’t have a clue what was behind it:

![32CE6ACA-B928-458C-BEC0-D98A3C843DFB](https://user-images.githubusercontent.com/194400/213595087-f8cba98e-924e-4391-9858-056f5fac1dd2.jpeg)

this was a worthwhile mini project. 🚀
